src/restapi/controlpanels/add.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from 'zod';
```

Comment:
ControlPanels cannot be "added" nor "deleted", some control panes have the ability to add and delete items (artefacts for their own purposes). We should create the proper actions. Adding it to the TODO list of remaining things.

Reply:
For now, let's leave it as this.